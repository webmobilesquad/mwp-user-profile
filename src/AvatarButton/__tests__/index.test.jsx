import render from 'preact-render-spy';
import { AvatarButton } from '../index';

describe('<AvatarButton />', () => {
  test('<AvatarButton /> matches snapshot', () => {
    expect(
      render(
        <AvatarButton
          src="https://mywebportal.michelin-solutions.com/msn-avatars/73c9c100a15a943fc0446f4b5e880d12.png"
          alt="John Doe"
        />,
      ),
    ).toMatchSnapshot();
  });
});
