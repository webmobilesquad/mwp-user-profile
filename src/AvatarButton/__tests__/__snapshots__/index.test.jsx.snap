// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AvatarButton /> <AvatarButton /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<button
  type="button"
  class="AvatarButton"
>
  <img
    src="https://mywebportal.michelin-solutions.com/msn-avatars/73c9c100a15a943fc0446f4b5e880d12.png"
    alt="John Doe"
   />
</button>

`;
